DataScience/dashboard_utils.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import argparse
 import time
-
+from loggers.logger_wrapper import Logger
```

Comment:
`logger_wrapper` is less than ideal as you're exposing class details to the caller. They don't need to know this is a wrapper. A preferable import is `from loggers import Logger`. This should be possible by adding `from logger_wrapper import Logger` to the loggers/__init__.py file.

Reply:
fixed